README.md

```diff
@@ -126,6 +127,10 @@ To stay in touch with PRQL:
 - [Jupyter magic](https://pyprql.readthedocs.io/en/latest/magic_readme.html) —
   run PRQL in Jupyter, either against a DB, or a Pandas DataFrame / CSV /
   Parquet file through DuckDB.
+- [qStudio](https://www.timestored.com/qstudio/prql-ide) is a PRQL IDE that
```

Comment:
I don't think this should be in the readme — it's not a native integration — it wouldn't be a good for someone to just check out PRQL since it requires downloading a binary and placing it in your path...

Reply:
OK. Fixed.